TheForceEngine/TFE_Settings/settings.h

```diff
@@ -205,6 +205,7 @@ struct TFE_Settings_Game
 	bool df_autorun = false;			// Run by default instead of walk.
 	bool df_crouchToggle = false;		// Use toggle instead of hold for crouch.
 	bool df_ignoreInfLimit = true;		// Ignore the vanilla INF limit.
+	bool df_solidWallFlagFix = false;	// Solid wall flag is enforced for collision with moving walls.
```

Comment:
If this doesn't break any mods, I would suggest defaulting it to true so people get the best experience.

Reply:
Done. I cannot think of how it could break any mods...